### DISCUSSION TITLE
No module named 'scipy.fftpack._fftpack'

### DISCUSSION THREAD

**Discussion by shizhx3** [ROBelgium/MSNoise#307] [I think there is a bug] [answered]
I followed the instruction document to install the msnoise package, but when i run the test command, error occured:

(base) szx@shizhuoxuandeMac-mini ~ % msnoise test
Traceback (most recent call last):
  File "/Users/szx/opt/anaconda3/bin/msnoise", line 5, in <module>
    from msnoise.scripts.msnoise import run
  File "/Users/szx/opt/anaconda3/lib/python3.9/site-packages/msnoise/scripts/msnoise.py", line 12, in <module>
    from ..api import connect, get_config, update_station, get_logger, get_job_types
  File "/Users/szx/opt/anaconda3/lib/python3.9/site-packages/msnoise/api.py", line 31, in <module>
    import scipy.fftpack._fftpack as sff
ModuleNotFoundError: No module named 'scipy.fftpack._fftpack'

**Comment by ThomasLecocq** [comment, marked as answer]
Hi,

please make use of the new version of msnoise: 

pip uninstall msnoise
pip install http://msnoise.org/master.zip

and use the following documentation : http://msnoise.org/doc/master/

**Reply by shizhx3** [reply]
Yeah, I noticed this when i want to solve this error. And I tried again this time. it seems to be a version number problem. Could you please check this ?
(base) szx@shizhuoxuandeMac-mini ~ % msnoise test
/Users/szx/opt/anaconda3/lib/python3.9/site-packages/msnoise/_version.py:153: UserWarning: msnoise could not determine its version number. Make sure it is properly installed. This for example happens when installing from a zip archive of the msnoise repository which is not a supported way of installing msnoise.
  warnings.warn("msnoise could not determine its version number. Make "
/Users/szx/opt/anaconda3/lib/python3.9/site-packages/msnoise/_version.py:153: UserWarning: msnoise could not determine its version number. Make sure it is properly installed. This for example happens when installing from a zip archive of the msnoise repository which is not a supported way of installing msnoise.
  warnings.warn("msnoise could not determine its version number. Make "
Usage: msnoise [OPTIONS] COMMAND [ARGS]...
Try 'msnoise --help' for help.

**Reply by ThomasLecocq** [reply]
as said, use the right documentation with the dev version

**Comment by ThomasLecocq** [comment]
the architecture of the commands has changed, the tests are now in "msnoise utils test"

**Comment by malmassri** [comment]
I am trying to run MSNoise but i receive ModuleNotFoundError: No module named 'scipy.fftpack._fftpack'

Any suggestions to solve this matter? 

Thanks

**Reply by ThomasLecocq** [reply]
did you try the above solutions?

**Reply by malmassri** [reply]
Yes, I did but still face the issue. I tried to install it in both windows and linux but i have no idea why is not working.

**Reply by ThomasLecocq** [reply]
the dev version doesn't have the culprit lines, make sure to remove the previous sources completely
